src/test/groovy/life/qbic/datamodel/dtos/business/ProductItemSpec.groovy

```diff
@@ -25,6 +27,50 @@ class ProductItemSpec extends Specification {
         productItem.product == product
     }
 
+    def "Equal ProductItems shall be equal"() {
+
+        given: "two product item contents"
+        def referenceProduct = new Sequencing("RNA Sequencing", "This package manages the pricing for all RNA sequencings", 1.0, 3.0, ProductUnit.PER_SAMPLE, 1, Facility.QBIC)
+        Double referenceQuantity = 1.0
+        Double referenceTotalPrice = 3.0
+        Double referenceQuantityDiscount = 0.0
+
+        when: "two product items are created from the same content"
+        ProductItem item1 = new ProductItem(referenceQuantity, referenceProduct, referenceTotalPrice, referenceQuantityDiscount)
+        ProductItem item2 = new ProductItem(referenceQuantity, referenceProduct, referenceTotalPrice, referenceQuantityDiscount)
+
+        ProductItem item3 = new ProductItem(referenceQuantity, referenceProduct)
+        ProductItem item4 = new ProductItem(referenceQuantity, referenceProduct)
+
+        then: "the product items are equal"
+        item1 == item2
+        item3 == item4
+    }
+
+    @Unroll
+    def "Different ProductItems shall be different for property #differentProperty"() {
+
+        given: "a reference product item and a different product"
+        def referenceProduct = new Sequencing("RNA Sequencing", "This package manages the pricing for all RNA sequencings", 1.0, 3.0, ProductUnit.PER_SAMPLE, 1, Facility.QBIC)
+        Double referenceQuantity = 1.0
+        Double referenceTotalPrice = 3.0
+        Double referenceQuantityDiscount = 0.0
+
+        when: "we create a product item with different information"
+        ProductItem reference = new ProductItem(referenceQuantity, referenceProduct, referenceTotalPrice, referenceQuantityDiscount)
+        ProductItem differentItem = new ProductItem(quantity, product, totalPrice, quantityDiscount)
+
+        then: "the product items are not equal"
+        reference != differentItem
+
+        where: "for every property"
+        differentProperty | quantity| product| totalPrice| quantityDiscount
+        "quantity" | 1.0 +1 | new Sequencing("RNA Sequencing", "This package manages the pricing for all RNA sequencings", 1.0, 3.0, ProductUnit.PER_SAMPLE, 1, Facility.QBIC) | 3.0 | 0.0
```

Comment:
Curiosity: whats the `+1` doing?
```suggestion
        "quantity" | 1.0 +1 | new Sequencing("RNA Sequencing", "This package manages the pricing for all RNA sequencings", 1.0, 3.0, ProductUnit.PER_SAMPLE, 1, Facility.QBIC) | 3.0 | 0.0
```

Reply:
the +1 indicates that it differs from the original one. I left it as `1.0 + 1` instead of `2.0` to highlight that something changed.